changelog.txt

```diff
@@ -1,7 +1,17 @@
-4.3.1
+4.3.2
 =============
+New Features:
+- **MSAL.NET now sends telemetry data to the /token endpoint in regards to the error code of the previous request, if applicable**. This will enable MSAL.NET to determine reliablity across public client application calls.
+
 Bug Fixes:
+- **For iOS 13, iOS broker may or may not return the source application**. The source appplication is used by MSAL.NET to verify the response is coming from broker. MSAL.NET will now also create a nonce to send in the broker request and will verify the same nonce is returned in the broker response in the case of a missing source application. [Issue](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/1375)
+- **During a silent call, a token was issued for a given user for the initial tenant, and not the specified tenant**. MSAL.NET now uses the specified tenant. [Issue](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/1365)
```

Comment:
After MSAL.NET acquired a token for a user, and the user signed-out - remove account, MSAL.NET was attempting to acquire the token with the same tenant as the first account, instead of using the tenant specified in the authority when building the application.